fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -787,6 +787,25 @@ public List<Long> getComputeNodeIds(boolean needAlive) {
         }
     }
 
+    public int getDecommissionedComputeNodeNumber() {
+        return getDecommissionedComputeNodeIds().size();
+    }
+
+    public List<Long> getDecommissionedComputeNodeIds() {
+        List<Long> computeNodeIds = Lists.newArrayList(idToComputeNodeRef.keySet());
+
+        Iterator<Long> iter = computeNodeIds.iterator();
+        while (iter.hasNext()) {
+            ComputeNode computeNode = this.getComputeNode(iter.next());
+            if (computeNode == null || !computeNode.isDecommissioned()) {
```

Comment:
I suppose this is just for interface compatibility, right now `decommission` statement doesn't support decommissioning a compute node, is it correct?

Reply:
it supposed to just drop cn, and no longer need operate decommission. I would like to remove this. :)

Reply:
ok. actually we are planning to support cn decommission as well in near feature. lol.

it is ok for now.